Polyfills/URL/Source/URLSearchParams.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <Babylon/JsRuntimeScheduler.h>
+#include <napi/napi.h>
+#include <unordered_map>
+#include <vector>
+
+namespace Babylon::Polyfills::Internal
+{
+    class URLSearchParams final : public Napi::ObjectWrap<URLSearchParams>
+    {
+    public:
+        static void Initialize(Napi::Env env);
+        explicit URLSearchParams(const Napi::CallbackInfo& info);
+        static constexpr auto JS_URL_SEARCH_PARAMS_CONSTRUCTOR_NAME = "URLSearchParams";
+        std::string GetAllParams();
+
+    private:
+        void Set(const Napi::CallbackInfo& info);
+        Napi::Value Has(const Napi::CallbackInfo& info);
+        Napi::Value Get(const Napi::CallbackInfo& info);
+        void parseKeyVal(const std::string& subStr);
+        void parseInputQueryStr(const std::string& queryStr);
```

Comment:
Function names should be upper camel case.

```suggestion
        void ParseKeyVal(const std::string& subStr);
        void ParseInputQueryStr(const std::string& queryStr);
```

Reply:
I still see lower camel case?

Reply:
Sorry, didn't save the files before the commit. Fixed now